modules/darwin/src/main/ts/ephox/darwin/api/InputHandlers.ts

```diff
@@ -24,9 +24,10 @@ const mouse = function (win: Window, container: Element, isRoot: (e: Element) =>
   const handlers = MouseSelection(bridge, container, isRoot, annotations);
 
   return {
-    mousedown: handlers.mousedown,
-    mouseover: handlers.mouseover,
-    mouseup: handlers.mouseup
+    clearState: handlers.clearState,
```

Comment:
Not sure I like having to do a `major` bump just to change this. So I'm going to request this be reverted please.

Reply:
Done